### DISCUSSION TITLE
NocoDB on Racknerd [Mar 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5373] [Discord Threads]
> 🙋‍♂️ **spaceb0t**

**Comment by o1lab** [comment]
> 🙋‍♂️ **dev_dipper**  
#### ok great. yes I'm gonna go with ubuntu 22.04 then install Docker > EasyPanel > nocoDB https://easypanel.io/docs   
#### Does the the specs I described  before ( 2cpu, 2.5GB ram...) are enough to run 1 or 2 websites (low traffic), nocodb, n8n and Nextcloud (family use) and few other dockers (low traffic mostly to test) to start?   
#### thanks <@407581134021722122>  
#  
> 🙋‍♂️ **spaceb0t**  
#### Yeah, should work   
####    
#### Just note that my PR for the NocoDB template in EasyPanel just got merged a couple of days ago and won’t be live until the next release   
####    
#### I’m not the project developer, but I imagine he’ll push a release with the new templates in the next couple of days   
####    
####    
#### Here’s a link to the PR: https://github.com/easypanel-io/templates/pull/217  
#  
> 🙋‍♂️ **dev_dipper**  
#### Ok perfect, thank you so much man 🙂 I'll keep you posted with my progress. take care  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1075738387576795137)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
